src/main/java/com/sap/sgs/phosphor/fosstars/model/qa/AbstractVerification.java

```diff
@@ -108,7 +108,7 @@
 
         String label = record.get("label");
         Label expectedLabel = null;
-        if (!"None".equals(label) && !label.isEmpty() && labelParser != null) {
+        if (!"None".equals(label) && !label.isEmpty()) {
```

Comment:
Will it be a good idea to extract this "None" value to a global constant and use it everywhere?  Just to counter the fact that spelling differences, case sensitivity etc?

Reply:
The constant is currently used only in the `AbstractVerification` class. If we start using this constant anywhere else, we'll definitely extract it to a final static field.

Reply:
Makes sense, how about changing it to equalsIgnoreCase ?

Reply:
Good point.